### DISCUSSION TITLE
Concurrent production deployments

### DISCUSSION THREAD

**Discussion by astanciu** [vercel/community#631] [Help] [answered]
Hi, is possible to do the following:

- Vercel project, does not use github integration, deployments happen manually or through github actions
- Currently, we have a production deployment with a production domain, `example.com`
- Want to launch a new version of the app, with production env vars, but w/o taking over the production domain (example.com).. aka a shadow deployment. 

The documentation states that when you do a production deployment (ie, using the --prod arg), it will assign the production domain.

How can I do this? How an I get a build that uses the Production env variables, but w/o a domain alias, or an alias that I can choose myself (ex: new.example.com)?

**Comment by amyegan** [comment, marked as answer]
Hi @astanciu. It sounds like a staging environment would be a good solution. You can set your production environment variables to be used on a specific preview branch and set a domain for the branch. Here’s a support article outlining the process [https://vercel.com/support/articles/set-up-a-staging-environment-on-vercel](https://vercel.com/support/articles/set-up-a-staging-environment-on-vercel)
